src/lib/utils.ts

```diff
@@ -670,12 +684,16 @@ export function patchIPFSUrlsIfApplicable(
   return receivedData;
 }
 
-export function countNumberOfBytes(data: string) {
-  return stripHexPrefix(data).length / 2;
+export function countNumberOfBytes(data: Hex) {
+  if (data == null) {
+    return 0;
+  }
+
+  return data.slice(2).length / 2;
 }
 
-export function countSignificantBits(data: string) {
-  const bytes = hexToBytes(data);
+export function countSignificantBits(data: Hex) {
+  const bytes = hexToBytes(data as Hex);
```

Comment:
This shouldn't be necessary, since it is the parameter type no?

```suggestion
  const bytes = hexToBytes(data);
```

Reply:
True